foqus_lib/core.py

```diff
@@ -0,0 +1,211 @@
+###############################################################################
+# FOQUS Copyright (c) 2012 - 2021, by the software owners: Oak Ridge Institute
```

Comment:
Make sure this is the new copyright header, with at least this year in the range.